src/builders/node-view-builder.js

```diff
@@ -44,19 +44,19 @@ export function nodeViewBuilder(node) {
       view.setAttribute("data-lynx-name", node.spec.name);
       var fragmentComponent = "#" + node.spec.name;
       if (node.base) {
-        view.setAttribute("data-jsua-view-uri", url.resolve(node.base, fragmentComponent));
+        view.setAttribute("data-jsua-view-uri", util.resolveUrlForNode(node, fragmentComponent));
       }
     }
 
-    if (hasScope(node)) view.setAttribute("data-lynx-scope", node.value.scope);
+    if (hasScope(node)) view.setAttribute("data-lynx-scope", util.resolveUrlForNode(node, node.value.scope));
```

Comment:
Consider moving the relative URI resolution into the view builders for the `content` objects as that seems like the most appropriate place for the knowledge of the `scope` value space to reside. It seems like this line should continue to do what it was doing before.